appengine/building-an-app/build/server.js

```diff
@@ -14,7 +14,7 @@
 
```

Comment:
Update copyright year to this year ☝🏼  https://github.com/GoogleCloudPlatform/nodejs-docs-samples/pull/3617/checks?check_run_id=21334109681

Reply:
The problem was with the Google Inc versus Google LLC